datasets/gov/lsd/sveikata/diagnozes/diagnozes.csv

```diff
@@ -0,0 +1,39 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/lsd/sveikata/diagnozes,,,,,,,,,,,,Diagnozių dažniai,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,EspbiDiagnozes025,,,vda_id,,,4,,,Diagnozių dažniai pagal E025 formos duomenis,"""Diagnozių TLK-10-AM kodų, nurodytų ESPBI IS E025 dokumente (“Ambulatorinio apsilankymo aprašymas”), dažniai pagal apsilankymo metus, lytį ir savivaldybę sveikatos priežiūros įstaigos (SPĮ), kurioje nustatyta diagnozė. Nurodytas skaičius unikalių asmenų, kuriems per ataskaitinius metus E025 dokumente diagnozės laukelyje buvo bent kartą įrašytas konkretus TLK ""kodas, ir šių asmenų amžiaus vidurkis apsilankymų SPĮ metu."
```

Comment:
Modelio pavadinimas turėtų būti vienaskaita. Šiuo atveju, panašiau, kad duomenų subjektas / objektas yra diagnozės dažnis.

Reply:
Kadangi lentelėse pateikiama informacija ne apie vieną diagnozę, tai manau tikslinga palikti "diagnozių dažnis"

Reply:
tada gal ir modelio pavadinimas "EspbiDiagnozesDaznis025" turėtų būti (kažką gal galima sutrumpinti iš šitų žodžių).

Reply:
ne taip supratau, čia kalbam apie lentelės pavadinimą, tada taip, labai išilgėja. ESPBI reikalinga, nes nurodo iš kokios sistemos paimti duomenys, 025 - medicininė forma. Pavadinimą palikčiau koks buvo, aprašymuose pakeičiau dažniai į dažnis

Reply:
Modelio pavadinimas turi atitikti duomenų subjekto / objekto (t.y. vienos duomenų eilutės) pavadinimą, šiuo atveju, jeigu gerai suprantu, tai būtų dažnis diagnozės. Bet gal galima pailkti ir taip.